docs/topics/crud-mission-intro.adoc

```diff
@@ -14,6 +14,7 @@ The booster application exposes an HTTP API, which provides endpoints that allow
 * Execute an HTTP `DELETE` request on the `api/fruits/*` endpoint, passing in a valid ID as an argument. This removes the item with the specified ID from the database and returns an HTTP code `204` (No Content) as a response. If you pass in an invalid ID, the call results in an HTTP error `404`.
 
 ifndef::http-api-nodejs[]
+.Integration Tests and Design Tradeoffs
```

Comment:
One, I'm not really sure what is the value of this heading, as "integration tests" and "design tradeoffs" are so different they shouldn't really be under one heading. Two, the heading will only be shown in "not Node.js" situation -- in the Node.js rendering, there will be no heading and the "design tradeoffs" paragraph will be under the previous heading anyway.

Reply:
I'll restore the headings, keep them separate for "Integration Tests"  and "Design Tradeoffs"